code/go/0chain.net/chaincore/chain/state/state_context.go

```diff
@@ -60,6 +60,8 @@ type CommonStateContextI interface {
 	GetTrieNode(key datastore.Key, v util.MPTSerializable) error
 	GetBlock() *block.Block
 	GetLatestFinalizedBlock() *block.Block
+	GetConfig() *storagescConfig
```

Comment:
The config could be different in different SCs, such as minersc, storagesc. So the `storagescConfig` may only work for storagesc. The issue of #1392 is not an easy task actually, we will need to identify what config are common global config from 0chain.yaml, and what need to store in separate SC packages. We will also need to maintain in memory config variables for them as caches so that we don't need load them from MPT each time when we need to read from them.

Reply:
add mapped configs by sc to the context and it might work